modules/nw-cluster-mtu-change.adoc

```diff
@@ -0,0 +1,482 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-cluster-network-mtu.adoc
+
+[id="nw-cluster-mtu-change_{context}"]
+= Changing the cluster MTU
+
+As a cluster administrator, you can change the maximum transmission unit (MTU) for your cluster. The migration is disruptive and nodes in your cluster might be temporarily unavailable as the MTU update rolls out.
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You are logged in to the cluster with a user with `cluster-admin` privileges.
+* You identified the target MTU for your cluster. The correct MTU varies depending on the cluster network provider that your cluster uses:
+** *OVN-Kubernetes*: The cluster MTU must be set to `100` less than the lowest MTU value in your cluster.
+** *OpenShift SDN*: The cluster MTU must be set to `50` less than the lowest MTU value in your cluster.
+
+.Procedure
+
+To increase or decrease the MTU for the cluster network complete the following procedure.
+
+. To obtain the current MTU for the cluster network, enter the following command:
++
+[source,terminal]
+----
+$ oc describe network.config cluster
+----
++
+.Example output
+[source,text]
+----
+...
+Status:
+  Cluster Network:
+    Cidr:               10.217.0.0/22
+    Host Prefix:        23
+  Cluster Network MTU:  1400
+  Network Type:         OpenShiftSDN
+  Service Network:
+    10.217.4.0/23
+...
+----
+
+. Prepare your configuration for the hardware MTU:
+
+** If your hardware MTU is specified in a NetworkManager connection configuration, complete the following steps. This approach is the default for {product-title} if you do not explicitly specify your network configuration with DHCP, a kernel command line, or some other method. Your cluster nodes must all use the same underlying network configuration for the following procedure to work unmodified.
+
+... To find the primary network interface, enter the following command:
++
+[source,terminal]
+----
+$ oc debug node/<node_name> -- chroot /host ip route list match 0.0.0.0/0 | awk '{print $5 }'
```

Comment:
I just noticed that for ovn-k, this would give you br-ex as interface name and that's not what we want. So this step instruction would only be valid for SDN.

For OVN-K, the interface we want is the system interface slaved on br-ex, several ways to get it:

`oc debug node/<node-name> -- chroot /host nmcli -g connection.interface-name c show ovs-if-phys0`

or

`oc debug node/<node-name> -- chroot /host ovs-vsctl --columns name find interface type=system | cut -d: -f2`

Reply:
@jcaamano, awesome, thank you!